django/contrib/admin/static/admin/css/base.css

```diff
@@ -104,7 +104,7 @@ body {
 /* LINKS */
 
 a:link, a:visited {
-    color: var(--link-fg);
+    color: var(--link-hover-color);
```

Comment:
It's covered by #16101 and ticket-34038. I think we should make `--link-fg` color darker to meet AA, not change to `--link-hover-color` (see [comment](https://github.com/django/django/pull/16101/files#r1022588796)).